app-misc/egads/egads-0.9.5-r2.ebuild

```diff
@@ -15,7 +15,17 @@ KEYWORDS="~amd64 ~ppc x86"
 
 EGADS_DATADIR=/var/run/egads
 
+PATCHES=(
+	"${FILESDIR}"/egads-0.9.5-make-build-work-with-clang16.patch
+)
+
+src_unpack() {
```

Comment:
Drop

Reply:
will do, I only put this in there because I tried out eaclocal and so on in there